src/components/ContextBar.js

```diff
@@ -90,17 +93,14 @@ export const ContextBar = ({ setDeletingWorkspace, setCloningWorkspace, setShari
       onSuccess: async () => {
         setCloudEnvOpen(false)
         await refreshRuntimes(true)
+        await refreshApps()
       },
       onDismiss: async () => {
         setCloudEnvOpen(false)
         await refreshRuntimes(true)
+        await refreshApps()
```

Comment:
This is the bug fix. We weren't refreshing apps before, causing the stale display.